### PR TITLE
Better error messages

### PR DESCRIPTION
**Pull request by magicdawn**
#1462 related

### CODE REVIEW

**Comment by dougwilson** [comment]
Hi @magicdawn can you update your PR to reflect the requested changes from that PR? That's what is holding up the PR, and this is just a clone, which isn't bringing it closer to land if the requested changes are not implemented.

**Comment by magicdawn** [comment]
Yeah Updating...

**Comment by magicdawn** [comment]
@dougwilson check on this

**Comment by magicdawn** [comment]
Oh I get it 😂

**Comment by magicdawn** [comment]
@dougwilson How about this

File: Readme.md [hunk 1]
```diff
@@ -1114,6 +1114,9 @@ object. Additionally they typically come with two extra properties:
 * `err.fatal`: Boolean, indicating if this error is terminal to the connection
   object. If the error is not from a MySQL protocol operation, this properly
   will not be defined.
+* `err.sql`: String, not enumerable, contains the SQL of the failed query.
+  This can we useful when using a higher level interface like an ORM
```
Comment (AdriVanHoudt) [inline Readme.md]: `can we` -> `can be`

**Comment by magicdawn** [comment]
Typo fixed

**Comment by magicdawn** [comment]
ping

**Comment by magicdawn** [comment]
ping ... @dougwilson

**Pull request closed without merging**